blueocean-personalization/src/main/js/components/FavoritePipeline.jsx

```diff
@@ -73,6 +74,11 @@ export class FavoritePipeline extends Component {
     }
 
     render() {
+        // TODO: this should probably key off a more generic capability like 'FavoritableItem'
+        if (capable(this.props.pipeline, 'hudson.matrix.MatrixProject')) {
+            return null;
+        }
+
```

Comment:
@michaelneale this might be worth discussing further. Maybe it's not a concern to be coding against specific job types, although it won't necessarily scale well if a new job type comes along that shouldn't be favoritable. Another idea is to add a capability that indicates the job can be favorited, and returning null if that capability is not present.